### ISSUE TITLE
Exclusions? (by IP or User Agent)

### ISSUE THREAD

**Issue by nuclearspike** [nfriedly/express-rate-limit#28]
Is there any way to specify to ignore certain user-agents or ips?

After I installed this and set it for normal usage, it ended up failing for newrelic and my load tester sites and I'm looking for a way to white-list user agents or IPs to prevent load testing or new relic from being subject to being rate limited.

**Comment by bogini** [comment]
@nuclearspike Check this out

https://github.com/nfriedly/express-rate-limit/pull/29

You'd be able to do:

```
let whitelisted = ['127.0.0.1'];

new RateLimit({
  skip: function (req, res) {
    return whitelisted.indexOf(req.ip) !== -1;
  }
})
```

**Comment by nfriedly** [comment]
Fixed by #29
